### DISCUSSION TITLE
`renv` and `Quarto` cumbersome setup?

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#855] [Q&A] [answered]
Hi,

Currently, it seems impossible de get relative path from the `.qmd` and `renv` library at the same time.
All has to work from root directory in order to work with `renv`[^1].

In other words, the only way to work with both is to:
1. write a `_quarto.yml` with `execute-dir: project`
2. use root-project relative paths everywhere

I was expecting a behaviour as the `RMarkdown` one for this.

Is there a way to stay at `.qmd` level, but get access to `renv` library?

[^1]: Related to issue raised in #794 and quarto-dev/quarto-vscode#15

**Comment by jjallaire** [comment, marked as answer]
This should be resolved here (https://github.com/quarto-dev/quarto-cli/commit/301a86aec43d98187cb24d6b3759714593b61cb5). That was a commit from ~ 10 days ago -- are you running the most recent version of Quarto?

The behavior should be that any `renv` within the project directory will automatically be utilized even with default `execute-dir` behavior. If you aren't seeing this pls. post a minimal reproducible example and I'll investigate further!

**Reply by mcanouil** [reply]
I opened an issue in `quarto-vscode` since after some testing, it (at least) appears to be how the `render` action button starts `quarto`.
quarto-dev/quarto-vscode#19

To note, the same behaviour might also be present in RStudio.

**Reply by jjallaire** [reply]
This should now be resolved (see discussion in quarto-vscode)

This isn't an issue in RStudio b/c it always inherits the renv from the currently active project.
